packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -16,6 +16,29 @@ String shortHash(Object object) {
 String describeIdentity(Object object) =>
     '${object.runtimeType}#${shortHash(object)}';
 
+/// Returns a short description of an enum value.
+///
+/// Strips off the enum class name from the `enumEntry.toString()`.
+///
+/// For example:
+///
+/// ```dart
+/// enum Day {
+///   monday, tuesday, wednesday, thursday, friday, saturday, sunday
+/// }
+///
+/// main() {
+///   assert(Day.monday.toString() == 'Day.monday');
+///   assert(describeEnum(Day.monday) == 'monday');
+/// }
+/// ```
+String describeEnum(Object enumEntry) {
```

Comment:
Can you file a bug on the Dart team saying that we have resorted to this pattern and that it would be nice if enum values just had a toString variant that did this?